docs/Custom_Device_Types.md

```diff
@@ -0,0 +1,184 @@
+## Custom Device Types
+
+While deployed to the Target, all custom devices run inside the VeriStand Engine. The engine is the non-visible mechanism that controls the timing of the entire system as well as communication between the Target and Host Computer. See **[VeriStand Manual](https://www.ni.com/documentation/en/veristand/latest/manual/manual-overview/)** » **[VeriStand Environment](https://www.ni.com/documentation/en/veristand/latest/manual/environment/)** » **[Components of a VeriStand Project](https://www.ni.com/documentation/en/veristand/latest/manual/project-components/)** » **[VeriStand Engine](https://www.ni.com/documentation/en/veristand/latest/manual/vs-engine/)** for more information.
```

Comment:
can you "non-visible"?  It doesn't add anything to the description, since you can 'see the engine' via the console and the effects the engine produces.